app/workers/process_photo.rb

```diff
@@ -15,6 +15,15 @@ def perform(id)
 
       photo.processed_image.store!(unprocessed_image)
 
+      ##unless current_user.strip_exif
+		  exifdata = MiniExiftool.new unprocessed_image.path, :coord_format => '%+.18f'
+
+		  photo.location =
+		  if exifdata.GPSLatitude and exifdata.GPSLongitude
+		     Location.new(:lat => exifdata.GPSLatitude, :lng => exifdata.GPSLongitude)
```

Comment:
- Tab detected.
- Use 2 (not 3) spaces for indentation.
- Use the new Ruby 1.9 hash syntax.